internal/controller/drplacementcontrol.go

```diff
@@ -1186,45 +1194,71 @@ func (d *DRPCInstance) readyToSwitchOver(homeCluster string, preferredCluster st
 		}
 	}
 	// Allow switch over when PV data is ready and the cluster data is protected
-	return d.isVRGConditionMet(homeCluster, VRGConditionTypeDataReady) &&
-		d.isVRGConditionMet(homeCluster, VRGConditionTypeClusterDataProtected)
+	dataReady, err := d.isVRGConditionMet(homeCluster, VRGConditionTypeDataReady)
+	if err != nil {
+		d.log.Info("readyToSwitchOver", "Error", err.Error())
```

Comment:
We can use d.log.Error() now to keep errors easy to find. log.Info() is good if you are sure that all possible errors are not really an error, but in this case why do we return an error?

Reply:
We do not return error here, we return "not ready to switch over"